AirCasting/BluetoothSession/ReconnectionController.swift

```diff
@@ -14,7 +14,7 @@ protocol ReconnectionController {
     var delegate: ReconnectionControllerDelegate? { get set }
 }
 
-class DefaultReconnectionController: ReconnectionController, BluetoothConnectionObserver {
+class DefaultReconnectionObserver: ReconnectionController, BluetoothConnectionObserver {
```

Comment:
I'll leave it to you, but I'm not sure about this new name. Usually the likes of `Observer` or `Delegate` protocols are secondary characteristics of a class, not their main responsibility. In other words - this classes main purpose is to be a `ReconnectionController` and the `BluetoothConnectionObserver` conformance is accidental - it needs to be one to do the job, but it's a detail.